libs/documentation/src/lib/components/formly-autocomplete/demos/basic/autocomplete-basic.component.ts

```diff
@@ -62,6 +62,14 @@ export class FormlyAutocompleteBasic implements OnInit {
     this.settings.selectionMode = SelectionMode.SINGLE;
     this.settings.autocompletePlaceHolderText = 'eg: Level 1';
     this.settings.debounceTime = 350;
+    this.settings.hideCloseIcon = true;
+  }
+
+  onModelChange() {
+    // To changes the close icon on modal change
+    if (this.model.filters.firstName[0].id !== "1") {
+      this.settings.hideCloseIcon = false
+    }
```

Comment:
Let's update the demo with an else statement here to turn this bool back to true. If the user selects Level 2, then selects Level 1, I think the close icon will still be visible currently.

Reply:
Else block added